print/libpaper/Portfile

```diff
@@ -1,55 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    rrthomas libpaper 2.1.1 v
+revision        0
+checksums       rmd160  32ad59966a5d5c06a99343562fe9b0008221b93b \
+                sha256  a4e1297b69b9fd1054ee7f5bcc55f4d56da152d41d2eabdf18727a9cddc1f402 \
+                size    1262975
+
+github.tarball_from releases
 
-name            libpaper
-set mainversion 1.1.28
-# Keep in case need again in the future
-#set nmuversion  5
-#set realversion ${mainversion}+nmu${nmuversion}
-#version         ${mainversion}.${nmuversion}
-set realversion ${mainversion}
-version         ${mainversion}
 categories      print
-license         GPL-2
+license         GPL-3
```

Comment:
I am not aware if binary distribution of dependents is hindered by specifying all of these licenses simultaneously. Maybe the libpaper 2 port does not need to offer the `paper` program, since it is not part of the current libpaper 1 port.

Reply:
Binaries are now removed in a `post-destroot` action.